ReShade.vcxproj

```diff
@@ -40,7 +40,7 @@
     <TargetName>ReShade32</TargetName>
   </PropertyGroup>
   <PropertyGroup Label="Configuration" Condition="'$(Platform)'=='x64'">
-    <TargetName>ReShade64</TargetName>
+    <TargetName>Reshade64</TargetName>
```

Comment:
Whoops.

Reply:
Made the change. Have renamed the target name to dxgi to ease the debug process. When reverted to the original name, did  not noticed the case.